apps/web/src/pages/Users/UserInformationPage/UserInformationPage.tsx

```diff
@@ -70,6 +72,15 @@ const UserInfo_Fragment = graphql(/* GraphQL */ `
       expiryDate
       notes
       suspendedAt
+      submittedAt
+      placedDepartment {
+        id
+        departmentNumber
+        name {
+          en
+          fr
+        }
+      }
```

Comment:
My attempt at verbose typing let me catch what I needed to add 🧐 
Otherwise it is silent

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/cd57128d-02a7-4f88-a09d-5a96c4c39b36)